apps/ledger-live-mobile/src/analytics/SegmentSetup.tsx

```diff
@@ -10,6 +11,8 @@ const SegmentSetup = (): null => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  useFlushInBackground();
```

Comment:
is your component always mounted even in background mode ?

Reply:
Yes it stays mounted